dsl_parser/tests/test_functions.py

```diff
@@ -719,6 +718,55 @@ def get_property_from_get_property(self):
         plan = prepare_deployment_plan(self.parse(yaml))
         self.assertEqual('secret', plan['nodes'][0]['properties']['b'])
 
+    def test_get_property_runtime(self):
```

Comment:
maybe add tests that runs get_prop that had inside get_secret and see that also works with the flag